package.json

```diff
@@ -168,6 +168,8 @@
     "underscore": "^1.13.1"
   },
   "devDependencies": {
+    "@dword-design/eslint-plugin-import-alias": "^4.0.8",
+    "@trivago/prettier-plugin-sort-imports": "^4.2.0",
```

Comment:
Follow up on this [discussion](https://expensify.slack.com/archives/C01GTK53T8Q/p1696838981418589?thread_ts=1696833372.663479&cid=C01GTK53T8Q). Let's try other libraries as well, example: `eslint-plugin-import`

Reply:
The reason why we're using this package: https://github.com/Expensify/App/pull/26605#issuecomment-1726723132

Reply:
For vscode users I recommend this config:
```
//.vscode.settings.json
"editor.codeActionsOnSave": {
    "source.fixAll.eslint": true
}
```

Thanks for the context!

Reply:
What about performance consideration? is it speed than the prettier imports? Last time I tried in my work it did slowed entire repo's eslint when tested against entire repo.

Reply:
I don't like auto fix on save. It breaks Ctrl+Z.
Personally I use Shift+Option+F once file change is done.

The idea is to fix all imports automatically after running command with --fix. No need to waste time fixing import orders one by one manually.